docker-compose-deploy.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  app:
+    build:
+      context: .
+    volumes:
+      - static_data:/vol/web
+    environment:
+      - SECRET_KEY=samplesecret123
+      - ALLOWED_HOSTS=127.0.0.1,localhost
```

Comment:
#this will make available to our localhost. use env file for production by using ip from aws